reframe/core/schedulers/slurm.py

```diff
@@ -236,11 +236,9 @@ def submit(self, job):
                 completed = _run_strict(cmd, timeout=self._submit_timeout)
                 break
             except SpawnedProcessError as e:
-                sbatch_error = '|'.join(self._resubmit_on_qos_errors)
-                if (
-                    not self._resubmit_on_qos_errors or
-                    not re.search(sbatch_error, e.stderr)
-                ):
+                sbatch_error = '|'.join(self._resubmit_on_errors)
+                if (not self._resubmit_on_errors or
+                    not re.search(sbatch_error, e.stderr)):
                     raise e
 
```

Comment:
I think we should log something here indicating the error condition that we encountered and that we are going to retry the submission in X seconds. Also, I don't remember whether we log the standard output and error of failing commands.